lumicks/pylake/detail/units.py

```diff
@@ -0,0 +1,37 @@
+
+au = "dimensionless"
```

Comment:
`arbitrary_unit != dimensionless` -- AU has a unit, it's just unknown to us.

Reply:
True. The variable name misrepresents the intent. I will change this.